src/page/Photo.tsx

```
@@ -116,6 +131,17 @@
           </div>
         </div>
       )}
+
+      {updateAvailable && (
+        <Button
+          variant="icon"
+          size="icon"
+          onClick={handleUpdate}
+          className="fixed bottom-4 right-4"
+        >
+          <DownloadIcon strokeWidth={1} size={16} />
```

Biome

[error] The DownloadIcon variable is undeclared.

By default, Biome recognizes browser and Node.js globals.
You can ignore more globals using the javascript.globals configuration.

(lint/correctness/noUndeclaredVariables)